### PR TITLE
Add support for setting static headers

### PR DESCRIPTION
**Pull request by sperand-io**
blocked by steps in https://github.com/segmentio/mongo-scripts/pull/129

### CODE REVIEW

**Comment by sperand-io** [comment]
@calvinfo mind reviewing quick?

File: lib/index.js [hunk 1]
```diff
@@ -106,20 +126,27 @@ function request(message, done){
...
     batch.push(function(done){
       var req = self
-        .post(hook)
+        .post(url)
         .type('json')
         .send(body)
         .parse(ignore);
 
+      for (var header in headers) {
+        req.set(header, headers[header])
```
Comment (calvinfo) [inline lib/index.js]: I think we can just do `req.set(headers)` and superagent will take care of this for us

File: test/index.js [hunk 2]
```diff
@@ -27,7 +27,15 @@ describe('Webhooks', function(){
 
   beforeEach(function(){
     settings = {
-      hooks: ['http://localhost:4000']
+      hooks: [{
```
Comment (calvinfo) [inline test/index.js]: can we add a test for a setting that's a string as well?

Comment (sperand-io) [inline test/index.js]: [L103](https://github.com/segment-integrations/integration-webhooks/pull/17/files#diff-910eb6f57886ca16c136101fb1699231R103) :)

**Comment by sperand-io** [comment]
@calvinfo addressed both comments — lgty otherwise?

File: test/index.js (hunk 2 above)
Comment (calvinfo) [inline test/index.js]: Ah cool, missed the other test!

**Comment by calvinfo** [comment]
lgtm!

**Pull request merged**